README.mkd

```diff
@@ -36,3 +36,8 @@ This above code output following.
 ```
 (let = s:message (printf "hello %d" (+ 1 (* 2 3))))
 ```
+
+## About project name
+
+We know "VimL" is not proper short form of "Vim script language".
```

Comment:
> We know "VimL" is not proper short form of "Vim script language".

We know a name "VimL" is not the common short form of "Vim script programming language" or "Vim scripting language."

主観的形容詞を排除してみました

Reply:
Where the name "Vim script programming language" came from?
Bram said the name is the Vim scripting language.

Reply:
Good point, let's only leave "Vim scripting language" instead of one with "programming."

I took that name translated from Japanese name `プログラミング言語Vim script` but Bram said it's a scripting language and not really a well formed (programming) language.

Reply:
@ujihisa @k-takata Took away a subjective adjective, how about it? 8bc42ca7b1ea18dffae7957128deaad793e7bd14

Reply:
> Bram said the name is the Vim scripting language.

oh, okay. fixed. f0cf638f1e6bdd0744f24586ee41cf5c58741d2a